types/token.go

```diff
@@ -89,7 +89,7 @@ const (
 	KIP20           TokenType = "KIP20"
 	APTOS           TokenType = "APTOS"
 	MOONBEAM        TokenType = "MOONBEAM"
-	KLAYTN          TokenType = "KLAYTN"
+	KLAYTN          TokenType = "KAIA"
```

Comment:
Don't forget to update the records in the db

Reply:
@iCell just for curiosity, which db are you talking about?

Reply:
i think he is talking about `backend-assets` db